shared/utilities/getTicketStatus/index.js

```diff
@@ -0,0 +1,127 @@
+import moment from 'moment';
+import { isBefore, isAfter, formatDate } from '../dateUtils';
+
+export function getActionLink(externalLinks, type) {
+  if (!externalLinks) return undefined;
+
+  return externalLinks.find((link) => {
+    return link.type === type;
+  });
+}
+
+/*
+ * isTicketPreRelease
+ * This function will return the correct values if the tickets are still to be released.
```

Comment:
It returns undefined if the predicate is not met. I can add that to each and every function comment but it seems pretty self explanatory.